docs/getting-started/supported-languages-and-tools.md

```diff
@@ -116,6 +116,13 @@ The table below lists all languages and frameworks that Codacy supports and the
       <td>-</td>
       <td>-</td>
     </tr>
+    <tr>
+      <td>Dart</td>
+      <td><a href="https://github.com/dart-lang/sdk/tree/master/pkg/analyzer_cli#dartanalyzer">dartanalyzer</a></td>
```

Comment:
Here I suggest that we don't include the URL fragment since we have a [periodic job that checks for broken links](https://github.com/codacy/docs/blob/master/.github/workflows/muffet.yml) and unfortunately [it doesn't work with URL fragments on GitHub](https://github.com/raviqqe/muffet/issues/52#issuecomment-542526533) because they're dynamically generated using JavaScript. :roll_eyes:

In summary, the current link will also generate a false positive.

```suggestion
      <td><a href="https://github.com/dart-lang/sdk/tree/master/pkg/analyzer_cli">dartanalyzer</a></td>
```